frigate/data_processing/common/license_plate/mixin.py

```diff
@@ -1396,12 +1396,7 @@ def lpr_process(
         )
 
         # don't overwrite sub label for objects that have a sub label
-        # that is not a license plate
-        if (
-            sub_label is not None
-            and obj_data.get("sub_label") is not None
-            and id not in self.detected_license_plates
-        ):
+        if sub_label is not None and obj_data.get("sub_label") is not None:
```

Comment:
This will stop a sub label from ever updating even if it was and is the same sub label but with a different score. I think this entire if can be removed per previous comments